src/NzbDrone.Core/Download/RejectedImportService.cs

```diff
@@ -32,7 +31,7 @@ public bool Process(TrackedDownload trackedDownload, ImportResult importResult)
 
         if (indexerSettings == null)
         {
-            trackedDownload.Warn(new TrackedDownloadStatusMessage(importResult.Errors.First(), new List<string>()));
+            trackedDownload.Warn(importResult.Errors.First());
```

Comment:
I think we could just send all the errors in the event there are multiple

Reply:
sending all results by creating the TrackedDownloadStatusMessage in this service.
Could also choose to overload trackedDownload.Warn() to accept a List<string> of errors?